OxfordBodleian/Juel-Jensen/BDLaethb6.xml

```diff
@@ -204,46 +8825,424 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <ref type="work" corresp="LIT4035SenkessarDL#Heda3SalAtnatewos"/> on <locus target="#60r"/>, <ref type="work" corresp="LIT4035SenkessarDL#Heda8EgziK"/> on <locus target="#64v"/>,
+                                        <ref type="work" corresp="LIT4035SenkessarDL#Heda14SalYohannes"/> on <locus target="#71r"/>, 
+                                        <ref type="work" corresp="LIT4035SenkessarDL#Heda17SalWasif"/> on <locus target="#74v"/> (added twice), and 
+                                        <ref type="work" corresp="LIT4035SenkessarDL#Tahs7Sal"/> on <locus target="#98r"/>. All but the first are salām-hymns that do not stand in a direct relationship with any of the 
+                                        commemorations of the day.
+                                    </desc>
+                                    <q xml:lang="gez"/>
+                                </item>
+                                
+                                
+                                <item xml:id="a3">
+                                    <locus target="#1r"/>
+                                    <desc type="Unclear">Colophon-type note written by <ref target="#h2"/> in the lower margin, stating that the manuscript was commissioned in the year of Grace 1744
+                                        by <persName ref="PRS5617IyasuII"/>.</desc>
+                                    <q xml:lang="gez">ለዝንቱ፡ መጽሐፈ፡ ስንክሳር፡ ዘአጽሐፎ፡ ንጉሠ፡ ነገሥት፡ ኢያሱ፡ ብርሃን፡ ሰገድ። ወስመ፡ አቡሁ፡ ንጉሠ፡ ነገሥት፡ በካፋ። ወስመ፡ እሙ፡ እቴጌ፡
+                                    ምንትዋብ፡ ከመ፡ ይኵኖ፡ መርሃ፡ ለመንግሥተ፡ ሰማያት። በዘመነ፡ ዮ<gap reason="illegible" unit="chars" quantity="1"/>ንስ፡ 
+                                        በ<date when-custom="1744" notBefore="1751" notAfter="1752"><hi rend="rubric" rendition="#partialRubric">፲፻</hi>ወ<hi rend="rubric" rendition="#partialRubric">፯፻፵</hi>ወ<hi rend="rubric" rendition="#partialRubric">፬</hi></date>
+                                            ዓመ<gap reason="illegible"/></q>
+                                </item>
+                                
+                                <item xml:id="a4">
+                                    <locus target="#186v"/>
+                                    <desc type="Unclear">Colophon-type note written written by <ref target="#h2"/> in the lower margin, stating that the manuscript was written in the year of Grace 1744
+                                        at the time of <persName ref="PRS5617IyasuII"/>, and indicating the name of the scribe as <persName ref="PRS13549SardaD"/>.</desc>
+                                    <q xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ አሜን።</hi>
+                                        ተጽሕፈ፡ ዝንቱ፡ መጽሐፈ፡ ስንክሣር<hi rend="rubric" rendition="#partialRubric">፥</hi> ዜና፡ ልደቶሙ፡ ወተዝካረ፡ እፈረቶሙ፡ ለአበዊነ፡ ቅዱሳን፡ 
```

Comment:
I would mark up እፈረቶሙ፡ with `sic` or I would correct it:  the expected reading is እረፍቶሙ፡ ("their rest", i.e. death), and an exchange of letters has occurred here.